hazelcast/src/test/java/com/hazelcast/scheduledexecutor/ScheduledExecutorplitBrainTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.scheduledexecutor;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.Member;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.SplitBrainTestSupport;
+import com.hazelcast.test.TestEnvironment;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+import static com.hazelcast.scheduledexecutor.TaskUtils.named;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.core.AllOf.allOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class ScheduledExecutorplitBrainTest
+        extends SplitBrainTestSupport {
+
+    private final String name = randomString();
+    private final int initialCount = 100;
+    private final int finalCount = initialCount + 50;
+
+    private Map<String, IScheduledFuture<Double>> allScheduledFutures = new HashMap<String, IScheduledFuture<Double>>();
+
+    static {
+        System.setProperty("hazelcast.test.use.network", "false");
```

Comment:
This looks wrong. The instances provided via `SplitBrainTestSupport` should already use the mock network.